scripts/docs.sh

```diff
@@ -31,3 +31,5 @@ cd docs
 pip3 install -r requirements.txt
 sphinx-build -nW -b html -d _build/doctrees . _build/html
 cd ..
+chmod +x scripts/update_robots.sh
```

Comment:
The CI job `b_docs` was failing because of `scripts/update_robots.sh: Permission denied` that's the reason why I added that.

Reply:
Probably because you did not commit the file as executable.

BTW, are you using Windows by chance? It does not support these flags so when using it you have to manually mark scripts as executable with `git add --chmod=+x <script path>`.

Reply:
Ah, I don't know about the the flag `--chmod=+x`, I will do that. I use Debian Linux.

Reply:
On Debian you should not have to use it (though I think it will work too if you do). It should be enough to run `chmod +x scripts/update_robots.sh` on your local machine once, before you commit the script. Then the file will be committed as executable and you won't have to run `chmod` in `docs.sh`.